src/minizinc/CLI/driver.py

```diff
@@ -149,6 +150,18 @@ async def create_process(
             solver (Union[str, Path, None]): Solver configuration string
                 guaranteed by the user to be valid until the process has ended.
         """
+        kwargs = {
+            "stdin": None,
+            "stdout": PIPE,
+            "stderr": PIPE,
+        }
+        if platform.system() == "Windows":
+            # On Windows, MiniZinc needs to be in its own console (but also hidden)
+            kwargs["startupinfo"] = subprocess.STARTUPINFO(
```

Comment:
`kwargs` might not be the best name for these arguments. Maybe we should call them `windows_spawn_options` (or something else descriptive).